k8s/bases/cse-valid-d2cc/data-bridge/cse-valid-d2cc-data-bridge-configmap.yaml

```diff
@@ -0,0 +1,69 @@
+data:
+  application.yml: |-
+    data-bridge:
+      configuration:
+        zone-id: "Europe/Paris"
+        target-process: CSE_VALID_D2CC
+        files:
+          - file-type: CGM
+            file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_(CO|CA)_Final(_Transit)?_CSE[0-9]{1,2}.uct"
+            time-validity: hourly
+            remote-file-regex: ".*CCC-CGMSel.*|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_(CO|CA)_Final(_Transit)?_CSE[0-9]{1,2}.uct"
+            source-directory: ""
+            sink-directory: "/CGMs"
+            file-list-persistence-file: /data/gridcapa/cse-valid-d2cc/ftp-metadata-store.cgm.properties
+          - file-type: EXPORT_CRAC
+            file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_Validation_Transit_FR[0-9]{1,2}.(xml|XML)"
+            time-validity: hourly
+            remote-file-regex: "CRAC_2D_CSE_Validation_Transit_[0-9]{8}V[0-9]{2}.zip|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_Validation_Transit_FR[0-9]{1,2}.(xml|XML)"
+            source-directory: ""
+            sink-directory: "/EXPORT_CRACs"
+            file-list-persistence-file: /data/gridcapa/cse-valid-d2cc/ftp-metadata-store.crac-export.properties
+          - file-type: IMPORT_CRAC
+            file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_FR[0-9]{1,2}.xml"
+            time-validity: hourly
+            remote-file-regex: "CRAC_2D_CSE_Validation_[0-9]{8}V[0-9]{2}.zip|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_FR[0-9]{1,2}.xml"
+            source-directory: ""
+            sink-directory: "/IMPORT_CRACs"
+            file-list-persistence-file: /data/gridcapa/cse-valid-d2cc/ftp-metadata-store.crac-import.properties
+          - file-type: GLSK
+            file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_(CO|CA)_GSK_CSE[0-9]{1,2}.xml"
+            time-validity: hourly
+            remote-file-regex: ".*CCC-GSKSel.*|(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_(CO|CA)_GSK_CSE[0-9]{1,2}.xml"
+            source-directory: ""
+            sink-directory: "/GLSKs"
+            file-list-persistence-file: /data/gridcapa/cse-valid-d2cc/ftp-metadata-store.glsk.properties
+          - file-type: TTC_ADJUSTMENT
+            file-regex: "TTC_Adjustment_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_2D[1-7]_Final_CSE[0-9]{1,2}.xml"
+            time-validity: daily
+            remote-file-regex: ".*CCC-TTCAdj-Final.*|TTC_Adjustment_(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_2D[1-7]_Final_CSE[0-9]{1,2}.xml"
+            source-directory: ""
```

Comment:
Source directory should be either "ecp" or "cia" depending on the filetype, but it cannot be identical for all filetypes.